bin/build-code

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+jspm build \
+	single-spa \
+	dist/single-spa.js \
+	--format umd \
+	--global-name singleSpa \
```

Comment:
umd with global-name makes the distributable usable via <script> tag, npm, and systemjs.